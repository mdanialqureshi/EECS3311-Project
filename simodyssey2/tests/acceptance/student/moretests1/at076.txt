-- test long no possible condition
test(1,1,1,1,30)
move(N)
move(W)
move(NW)
land
move(N)
status
wormhole
move(NW)
liftoff
move(NW)
land
move(E)
land
status
pass
pass
pass
pass
pass
pass
pass
pass
land
liftoff
land
