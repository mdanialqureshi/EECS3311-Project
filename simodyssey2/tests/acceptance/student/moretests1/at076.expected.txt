  state:0.0, ok
  Welcome! Try test(1,1,1,1,30)
->test(1,1,1,1,30)
  state:1.0, mode:test(1,1,1,1,30), ok
  Movement:none
    (1:1)  (1:2)  (1:3)  (1:4)  (1:5)  
    E---   *---   P---   ----   PP--   
    (2:1)  (2:2)  (2:3)  (2:4)  (2:5)  
    ----   ----   W---   ----   P---   
    (3:1)  (3:2)  (3:3)  (3:4)  (3:5)  
    PY--   ----   O---   PW--   ----   
    (4:1)  (4:2)  (4:3)  (4:4)  (4:5)  
    W---   P*--   P---   P*--   ----   
    (5:1)  (5:2)  (5:3)  (5:4)  (5:5)  
    PW--   ----   *---   ----   PW--   
->move(N)
  state:2.0, mode:test(1,1,1,1,30), ok
  Movement:
    [0,E]:[1,1,1]->[5,1,3]
    [3,P]:[1,5,2]->[5,4,1]
    [8,P]:[4,3,1]->[3,2,1]
    (1:1)  (1:2)  (1:3)  (1:4)  (1:5)  
    ----   *---   P---   ----   P---   
    (2:1)  (2:2)  (2:3)  (2:4)  (2:5)  
    ----   ----   W---   ----   P---   
    (3:1)  (3:2)  (3:3)  (3:4)  (3:5)  
    PY--   P---   O---   PW--   ----   
    (4:1)  (4:2)  (4:3)  (4:4)  (4:5)  
    W---   P*--   ----   P*--   ----   
    (5:1)  (5:2)  (5:3)  (5:4)  (5:5)  
    PWE-   ----   *---   P---   PW--   
->move(W)
  state:3.0, mode:test(1,1,1,1,30), ok
  Movement:
    [0,E]:[5,1,3]->[5,5,3]
    [8,P]:[3,2,1]->[4,1,2]
    [10,P]:[5,1,1]->[1,2,2]
    (1:1)  (1:2)  (1:3)  (1:4)  (1:5)  
    ----   *P--   P---   ----   P---   
    (2:1)  (2:2)  (2:3)  (2:4)  (2:5)  
    ----   ----   W---   ----   P---   
    (3:1)  (3:2)  (3:3)  (3:4)  (3:5)  
    PY--   ----   O---   PW--   ----   
    (4:1)  (4:2)  (4:3)  (4:4)  (4:5)  
    WP--   P*--   ----   P*--   ----   
    (5:1)  (5:2)  (5:3)  (5:4)  (5:5)  
    -W--   ----   *---   P---   PWE-   
->move(NW)
  state:4.0, mode:test(1,1,1,1,30), ok
  Movement:
    [0,E]:[5,5,3]->[4,4,3]
    [1,P]:[1,3,1]->[2,4,1]
    [2,P]:[1,5,1]->[5,1,1]
    [4,P]:[2,5,1]->[2,1,1]
    [6,P]:[3,4,1]->[4,4,4]
    [11,P]:[5,5,1]->[5,4,2]
    (1:1)  (1:2)  (1:3)  (1:4)  (1:5)  
    ----   *P--   ----   ----   ----   
    (2:1)  (2:2)  (2:3)  (2:4)  (2:5)  
    P---   ----   W---   P---   ----   
    (3:1)  (3:2)  (3:3)  (3:4)  (3:5)  
    PY--   ----   O---   -W--   ----   
    (4:1)  (4:2)  (4:3)  (4:4)  (4:5)  
    WP--   P*--   ----   P*EP   ----   
    (5:1)  (5:2)  (5:3)  (5:4)  (5:5)  
    PW--   ----   *---   PP--   -W--   
->land
  state:4.1, mode:test(1,1,1,1,30), error
  Negative on that request:no yellow dwarf at Sector:4:4
->move(N)
  state:5.0, mode:test(1,1,1,1,30), ok
  Movement:
    [0,E]:[4,4,3]->[3,4,1]
    [1,P]:[2,4,1]->[1,3,1]
    [3,P]:[5,4,1]->[1,3,2]
    [4,P]:[2,1,1]->[2,5,1]
    [8,P]:[4,1,2]->[5,5,1]
    (1:1)  (1:2)  (1:3)  (1:4)  (1:5)  
    ----   *P--   PP--   ----   ----   
    (2:1)  (2:2)  (2:3)  (2:4)  (2:5)  
    ----   ----   W---   ----   P---   
    (3:1)  (3:2)  (3:3)  (3:4)  (3:5)  
    PY--   ----   O---   EW--   ----   
    (4:1)  (4:2)  (4:3)  (4:4)  (4:5)  
    W---   P*--   ----   P*-P   ----   
    (5:1)  (5:2)  (5:3)  (5:4)  (5:5)  
    PW--   ----   *---   -P--   PW--   
->status
  state:5.1, mode:test(1,1,1,1,30), ok
  Explorer status report:Travelling at cruise speed at [3,4,1]
  Life units left:3, Fuel units left:2
->wormhole
  state:6.0, mode:test(1,1,1,1,30), ok
  Movement:
    [0,E]:[3,4,1]->[5,3,2]
    [1,P]:[1,3,1]->[2,3,2]
    [3,P]:[1,3,2]->[2,2,1]
    [4,P]:[2,5,1]->[1,4,1]
    (1:1)  (1:2)  (1:3)  (1:4)  (1:5)  
    ----   *P--   ----   P---   ----   
    (2:1)  (2:2)  (2:3)  (2:4)  (2:5)  
    ----   P---   WP--   ----   ----   
    (3:1)  (3:2)  (3:3)  (3:4)  (3:5)  
    PY--   ----   O---   -W--   ----   
    (4:1)  (4:2)  (4:3)  (4:4)  (4:5)  
    W---   P*--   ----   P*-P   ----   
    (5:1)  (5:2)  (5:3)  (5:4)  (5:5)  
    PW--   ----   *E--   -P--   PW--   
->move(NW)
  state:7.0, mode:test(1,1,1,1,30), ok
  Movement:
    [0,E]:[5,3,2]->[4,2,3]
    [1,P]:[2,3,2]->[1,4,2]
    [2,P]:[5,1,1]->[4,5,1]
    [3,P]:[2,2,1]->[1,2,3]
    [8,P]:[5,5,1]->[5,4,1]
    [11,P]:[5,4,2]->[4,5,2]
    (1:1)  (1:2)  (1:3)  (1:4)  (1:5)  
    ----   *PP-   ----   PP--   ----   
    (2:1)  (2:2)  (2:3)  (2:4)  (2:5)  
    ----   ----   W---   ----   ----   
    (3:1)  (3:2)  (3:3)  (3:4)  (3:5)  
    PY--   ----   O---   -W--   ----   
    (4:1)  (4:2)  (4:3)  (4:4)  (4:5)  
    W---   P*E-   ----   P*-P   PP--   
    (5:1)  (5:2)  (5:3)  (5:4)  (5:5)  
    -W--   ----   *---   P---   -W--   
->liftoff
  state:7.1, mode:test(1,1,1,1,30), error
  Negative on that request:you are not on a planet at Sector:4:2
->move(NW)
  state:8.0, mode:test(1,1,1,1,30), ok
  Movement:
    [0,E]:[4,2,3]->[3,1,3]
    [4,P]:[1,4,1]->[2,3,2]
    [11,P]:[4,5,2]->[3,4,1]
    (1:1)  (1:2)  (1:3)  (1:4)  (1:5)  
    ----   *PP-   ----   -P--   ----   
    (2:1)  (2:2)  (2:3)  (2:4)  (2:5)  
    ----   ----   WP--   ----   ----   
    (3:1)  (3:2)  (3:3)  (3:4)  (3:5)  
    PYE-   ----   O---   PW--   ----   
    (4:1)  (4:2)  (4:3)  (4:4)  (4:5)  
    W---   P*--   ----   P*-P   P---   
    (5:1)  (5:2)  (5:3)  (5:4)  (5:5)  
    -W--   ----   *---   P---   -W--   
->land
  state:9.0, mode:test(1,1,1,1,30), ok
  Explorer found no life as we know it at Sector:3:1
  Movement:
    [1,P]:[1,4,2]->[1,3,1]
    (1:1)  (1:2)  (1:3)  (1:4)  (1:5)  
    ----   *PP-   P---   ----   ----   
    (2:1)  (2:2)  (2:3)  (2:4)  (2:5)  
    ----   ----   WP--   ----   ----   
    (3:1)  (3:2)  (3:3)  (3:4)  (3:5)  
    PYE-   ----   O---   PW--   ----   
    (4:1)  (4:2)  (4:3)  (4:4)  (4:5)  
    W---   P*--   ----   P*-P   P---   
    (5:1)  (5:2)  (5:3)  (5:4)  (5:5)  
    -W--   ----   *---   P---   -W--   
->move(E)
  state:9.1, mode:test(1,1,1,1,30), error
  Negative on that request:you are currently landed at Sector:3:1
->land
  state:9.2, mode:test(1,1,1,1,30), error
  Negative on that request:already landed on a planet at Sector:3:1
->status
  state:9.3, mode:test(1,1,1,1,30), ok
  Explorer status report:Stationary on planet surface at [3,1,3]
  Life units left:3, Fuel units left:3
->pass
  state:10.0, mode:test(1,1,1,1,30), ok
  Movement:
    [2,P]:[4,5,1]->[5,1,1]
    [4,P]:[2,3,2]->[3,4,3]
    [8,P]:[5,4,1]->[5,5,1]
    [11,P]:[3,4,1]->[2,5,1]
    (1:1)  (1:2)  (1:3)  (1:4)  (1:5)  
    ----   *PP-   P---   ----   ----   
    (2:1)  (2:2)  (2:3)  (2:4)  (2:5)  
    ----   ----   W---   ----   P---   
    (3:1)  (3:2)  (3:3)  (3:4)  (3:5)  
    PYE-   ----   O---   -WP-   ----   
    (4:1)  (4:2)  (4:3)  (4:4)  (4:5)  
    W---   P*--   ----   P*-P   ----   
    (5:1)  (5:2)  (5:3)  (5:4)  (5:5)  
    PW--   ----   *---   ----   PW--   
->pass
  state:11.0, mode:test(1,1,1,1,30), ok
  Movement:none
    (1:1)  (1:2)  (1:3)  (1:4)  (1:5)  
    ----   *PP-   P---   ----   ----   
    (2:1)  (2:2)  (2:3)  (2:4)  (2:5)  
    ----   ----   W---   ----   P---   
    (3:1)  (3:2)  (3:3)  (3:4)  (3:5)  
    PYE-   ----   O---   -WP-   ----   
    (4:1)  (4:2)  (4:3)  (4:4)  (4:5)  
    W---   P*--   ----   P*-P   ----   
    (5:1)  (5:2)  (5:3)  (5:4)  (5:5)  
    PW--   ----   *---   ----   PW--   
->pass
  state:12.0, mode:test(1,1,1,1,30), ok
  Movement:
    [1,P]:[1,3,1]->[5,2,1]
    [2,P]:[5,1,1]->[1,1,1]
    [4,P]:[3,4,3]->[3,3,2]
    [8,P]:[5,5,1]->[1,4,1]
    (1:1)  (1:2)  (1:3)  (1:4)  (1:5)  
    P---   *PP-   ----   P---   ----   
    (2:1)  (2:2)  (2:3)  (2:4)  (2:5)  
    ----   ----   W---   ----   P---   
    (3:1)  (3:2)  (3:3)  (3:4)  (3:5)  
    PYE-   ----   O---   -W--   ----   
    (4:1)  (4:2)  (4:3)  (4:4)  (4:5)  
    W---   P*--   ----   P*-P   ----   
    (5:1)  (5:2)  (5:3)  (5:4)  (5:5)  
    -W--   P---   *---   ----   -W--   
->pass
  state:13.0, mode:test(1,1,1,1,30), ok
  Movement:
    [2,P]:[1,1,1]->[5,2,2]
    [11,P]:[2,5,1]->[1,5,1]
    (1:1)  (1:2)  (1:3)  (1:4)  (1:5)  
    ----   *PP-   ----   P---   P---   
    (2:1)  (2:2)  (2:3)  (2:4)  (2:5)  
    ----   ----   W---   ----   ----   
    (3:1)  (3:2)  (3:3)  (3:4)  (3:5)  
    PYE-   ----   O---   -W--   ----   
    (4:1)  (4:2)  (4:3)  (4:4)  (4:5)  
    W---   P*--   ----   P*-P   ----   
    (5:1)  (5:2)  (5:3)  (5:4)  (5:5)  
    -W--   PP--   *---   ----   -W--   
->pass
  state:14.0, mode:test(1,1,1,1,30), ok
  Movement:
    [2,P]:[5,2,2]->[1,1,1]
    (1:1)  (1:2)  (1:3)  (1:4)  (1:5)  
    P---   *PP-   ----   P---   P---   
    (2:1)  (2:2)  (2:3)  (2:4)  (2:5)  
    ----   ----   W---   ----   ----   
    (3:1)  (3:2)  (3:3)  (3:4)  (3:5)  
    PYE-   ----   O---   -W--   ----   
    (4:1)  (4:2)  (4:3)  (4:4)  (4:5)  
    W---   P*--   ----   P*-P   ----   
    (5:1)  (5:2)  (5:3)  (5:4)  (5:5)  
    -W--   P---   *---   ----   -W--   
->pass
  state:15.0, mode:test(1,1,1,1,30), ok
  Movement:
    [1,P]:[5,2,1]->[1,1,2]
    [8,P]:[1,4,1]->[2,5,1]
    [11,P]:[1,5,1]->[1,1,3]
    (1:1)  (1:2)  (1:3)  (1:4)  (1:5)  
    PPP-   *PP-   ----   ----   ----   
    (2:1)  (2:2)  (2:3)  (2:4)  (2:5)  
    ----   ----   W---   ----   P---   
    (3:1)  (3:2)  (3:3)  (3:4)  (3:5)  
    PYE-   ----   O---   -W--   ----   
    (4:1)  (4:2)  (4:3)  (4:4)  (4:5)  
    W---   P*--   ----   P*-P   ----   
    (5:1)  (5:2)  (5:3)  (5:4)  (5:5)  
    -W--   ----   *---   ----   -W--   
->pass
  state:16.0, mode:test(1,1,1,1,30), ok
  Movement:
    [2,P]:[1,1,1]->[2,5,2]
    (1:1)  (1:2)  (1:3)  (1:4)  (1:5)  
    -PP-   *PP-   ----   ----   ----   
    (2:1)  (2:2)  (2:3)  (2:4)  (2:5)  
    ----   ----   W---   ----   PP--   
    (3:1)  (3:2)  (3:3)  (3:4)  (3:5)  
    PYE-   ----   O---   -W--   ----   
    (4:1)  (4:2)  (4:3)  (4:4)  (4:5)  
    W---   P*--   ----   P*-P   ----   
    (5:1)  (5:2)  (5:3)  (5:4)  (5:5)  
    -W--   ----   *---   ----   -W--   
->pass
  state:17.0, mode:test(1,1,1,1,30), ok
  Movement:
    [1,P]:[1,1,2]->[1,5,1]
    [8,P]:[2,5,1]->[2,4,1]
    (1:1)  (1:2)  (1:3)  (1:4)  (1:5)  
    --P-   *PP-   ----   ----   P---   
    (2:1)  (2:2)  (2:3)  (2:4)  (2:5)  
    ----   ----   W---   P---   -P--   
    (3:1)  (3:2)  (3:3)  (3:4)  (3:5)  
    PYE-   ----   O---   -W--   ----   
    (4:1)  (4:2)  (4:3)  (4:4)  (4:5)  
    W---   P*--   ----   P*-P   ----   
    (5:1)  (5:2)  (5:3)  (5:4)  (5:5)  
    -W--   ----   *---   ----   -W--   
->land
  state:17.1, mode:test(1,1,1,1,30), error
  Negative on that request:already landed on a planet at Sector:3:1
->liftoff
  state:18.0, mode:test(1,1,1,1,30), ok
  Explorer has lifted off from planet at Sector:3:1
  Movement:
    [1,P]:[1,5,1]->[5,4,1]
    [2,P]:[2,5,2]->[3,1,4]
    [11,P]:[1,1,3]->[5,1,1]
    (1:1)  (1:2)  (1:3)  (1:4)  (1:5)  
    ----   *PP-   ----   ----   ----   
    (2:1)  (2:2)  (2:3)  (2:4)  (2:5)  
    ----   ----   W---   P---   ----   
    (3:1)  (3:2)  (3:3)  (3:4)  (3:5)  
    PYEP   ----   O---   -W--   ----   
    (4:1)  (4:2)  (4:3)  (4:4)  (4:5)  
    W---   P*--   ----   P*-P   ----   
    (5:1)  (5:2)  (5:3)  (5:4)  (5:5)  
    PW--   ----   *---   P---   -W--   
->land
  state:19.0, mode:test(1,1,1,1,30), ok
  Explorer found no life as we know it at Sector:3:1
  Movement:
    [1,P]:[5,4,1]->[1,3,1]
    (1:1)  (1:2)  (1:3)  (1:4)  (1:5)  
    ----   *PP-   P---   ----   ----   
    (2:1)  (2:2)  (2:3)  (2:4)  (2:5)  
    ----   ----   W---   P---   ----   
    (3:1)  (3:2)  (3:3)  (3:4)  (3:5)  
    PYEP   ----   O---   -W--   ----   
    (4:1)  (4:2)  (4:3)  (4:4)  (4:5)  
    W---   P*--   ----   P*-P   ----   
    (5:1)  (5:2)  (5:3)  (5:4)  (5:5)  
    PW--   ----   *---   ----   -W--   
